### PR TITLE
 dist: move testrunner to pythonlibs as package 

### PR DESCRIPTION
**Pull request by smlng**
### Contribution description

As testrunner is moved to dist/pythonlibs which is exported
via PYTHONPATH, testrunner is found by all test scripts.

### Issues/PRs references

#9758

### CODE REVIEW

**Comment by aabadie** [comment]
Nice one. I like it.

File: tests/bench_msg_pingpong/tests/01-run.py [hunk 1]
```diff
@@ -16,6 +15,5 @@ def testfunc(child):
 
 
 if __name__ == "__main__":
-    sys.path.append(os.path.join(os.environ['RIOTTOOLS'], 'testrunner'))
```
Comment (aabadie) [inline tests/bench_msg_pingpong/tests/01-run.py]: The line below can now be moved with the other imports at the top of the script.

**Comment by smlng** [comment]
@aabadie please note this is based on #9758, maybe you can also have a look at that one, too ... THX!

File: tests/bench_msg_pingpong/tests/01-run.py (hunk 1 above)
Comment (smlng) [reply tests/bench_msg_pingpong/tests/01-run.py]: yes will do

**Comment by aabadie** [comment]
@smlng, I don't think removing `sys` is a good idea. See [this](https://stackoverflow.com/questions/6501121/difference-between-exit-and-sys-exit-in-python)

**Comment by smlng** [comment]
> I don't think removing sys I

was suggested by @cladmi, but reading the linked blog-entry it sound reasonable to use `sys.exit` in out context - so should I drop the change?

**Comment by cladmi** [comment]
> was suggested by @cladmi, but reading the linked blog-entry it sound reasonable to use sys.exit in out context - so should I drop the change?

My bad, I did not know about this https://docs.python.org/3/library/constants.html#exit and never had issues before so was wondering why.

**Comment by smlng** [comment]
dropped the removal of `import sys` and fixed flake8 errors reported by Murdock, good to review this one again - needs squashing now. @cladmi and @aabadie?

**Comment by smlng** [comment]
squashed, I'd still like to have an ACK by @cladmi on this one, too. So please don't merge right away, but leave him do it, thx.

**Comment by cladmi** [comment]
I will take care of reviewing after the release.

**Comment by cladmi** [comment]
Changes look good, I will re-run tests and see if something complains.

**Comment by cladmi** [comment]
Some gardening may need to be done after PRs still using `sys.append.path` are merged.
The tests may be found using:

    git grep sys.path.append

**Pull request merged**